psutil/tests/__init__.py

```diff
@@ -583,7 +583,11 @@ def reap_children(recursive=False):
     # Terminate children.
     if children:
         for p in children:
-            terminate(p, wait_timeout=None)
+            if MACOS and sys.version_info.major == 3 and\
+                    sys.version_info.minor == 8:
+                terminate(p, sig=signal.SIGKILL, wait_timeout=None)
+            else:
+                terminate(p, wait_timeout=None)
```

Comment:
Mmm this should not be necessary because 3 lines later we use SIGKILL for the processes which survived SIGTERM.

Reply:
Looks like build stuck after revert this change. I revert also change with timeout for simplify debug, but have no idea how to debug it.